ecosystem/typescript/sdk/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ All notable changes to the Aptos Node SDK will be captured in this file. This ch
 **Note:** The Aptos TS SDK does not follow semantic version while we are in active development. Instead, breaking changes will be announced with each devnet cut. Once we launch our mainnet, the SDK will follow semantic versioning closely.
 
 ## Unreleased
+- Move `TypeTagParser` from `builder_utils.ts` to `type_tag.ts`
```

Comment:
this should have been merged with the previous PR?
also, let's add this PR change into the changelog